docs/streams/developer-guide/dsl-api.html

```diff
@@ -3548,6 +3548,7 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
 clicksPerRegion.toStream.to(outputTopic)
               </pre>
               <p>A complete example of user-defined SerDes can be found in a test class within the library.</p>
+              <p>The auto-configure of user-defined SerDes would be done at KafkaStreams construction time.</p>
```

Comment:
We can add this sentence, but this is true for Scala already.

We should update this docs, too: https://kafka.apache.org/21/documentation/streams/developer-guide/datatypes.html#streams-developer-guide-serdes